### PR TITLE
Added random placeholder in page load

### PR DESCRIPTION
**Pull request by luketevl**
Fix issue #15

### CODE REVIEW

File: src/js/translator.js [hunk 1]
```diff
@@ -3,20 +3,42 @@ import json from '../library.json';
...
-
+  
   const textTranslate = (e) => { 
+    let text;
     text = $translator.value.toLowerCase();  
-    forEach(library, (value, key) => { 
-      const regex = new RegExp(removeAccent(key), 'gm'); 
-      text = text.replace(regex, value); 
-    }); 
+    if(text == ''){
```
Comment (LFeh) [inline src/js/translator.js]: `if (text === '') {`

Spacements and comparison

File: src/js/translator.js [hunk 2]
```diff
@@ -3,20 +3,42 @@ import json from '../library.json';
...
+      $translator.placeholder = 'Digitar texto...';
+      $result.value = 'Tradução'; 
+      return ;
```
Comment (LFeh) [inline src/js/translator.js]: `return;`

Spacement

File: src/js/translator.js [hunk 3]
```diff
@@ -3,20 +3,42 @@ import json from '../library.json';
...
+  const getRandomText  = () => {
+    const keys = Object.keys(json);
+    const phrase = keys[generateRandomKey(keys)];
+    $translator.placeholder = phrase;
+    $result.value = translate(json, phrase);
+  };
 
   $translator.addEventListener('keyup', (e) => debounce(textTranslate(), 600));
+  window.addEventListener('load', () => getRandomText());
+
```
Comment (LFeh) [inline src/js/translator.js]: Extra space

File: src/js/translator.js [hunk 4]
```diff
@@ -3,20 +3,42 @@ import json from '../library.json';
...
+    if(text == ''){
+      $translator.placeholder = 'Digitar texto...';
+      $result.value = 'Tradução'; 
+      return ; 
+    }
+    text = translate(json, text);
     $result.value = text || 'Tradução';  
   };
+  
+  const getRandomText  = () => {
```
Comment (LFeh) [inline src/js/translator.js]: `const getRandomText = () => {`

Extra space before `=`

File: src/js/translator.js (hunk 1 above)
Comment (luketevl) [reply src/js/translator.js]: fix

File: src/js/translator.js (hunk 2 above)
Comment (luketevl) [reply src/js/translator.js]: fix

File: src/js/translator.js (hunk 3 above)
Comment (luketevl) [reply src/js/translator.js]: fix

File: src/js/translator.js (hunk 4 above)
Comment (luketevl) [reply src/js/translator.js]: fix

**Comment by LFeh** [comment]
Awesome! 

Thanks @luketevl 👏 👏 👏 👏

**Pull request merged**